Cargo.toml

```diff
@@ -1,9 +1,9 @@
 [package]
 name = "base_emoji"
-version = "1.0.0"
+version = "1.0.1"
```

Comment:
Should be 1.1.0, but just remove this part, then I use [`cargo-releaes`](https://github.com/sunng87/cargo-release) to make it happen